vterm.el

```diff
@@ -276,6 +282,19 @@ If nil, never delay")
                                        unless (member key vterm-keymap-exceptions)
                                        collect key))))
 
+
+(define-minor-mode vterm-copy-mode
+  "Toggle vterm copy mode."
+  :group 'vterm
+  :lighter " VtermCopy"
+  (if vterm-copy-mode
+      (progn                            ;enable vterm-copy-mode
+        (use-local-map vterm-copy-map)
+        (vterm-send-key "s" nil nil t))
+    (use-local-map vterm-raw-map)
+    (vterm-send-key "q" nil nil t)))
```

Comment:
For this to work you have to add `ixon` to the stty call after the sane here: https://github.com/akermu/emacs-libvterm/blob/ee07b5a1e798a8b45c242de2bcdabc009ff055be/vterm.el#L235

Reply:
done